tests/ops/integration_tests/saas/test_mailchimp_transactional_task.py

```diff
@@ -0,0 +1,214 @@
+import random
+from unittest import mock
+from uuid import uuid4
+
+import pytest
+
+from fides.api.ops.models.policy import ActionType
+from fides.api.ops.models.privacy_request import (
+    ExecutionLog,
+    ExecutionLogStatus,
+    PrivacyRequest,
+)
+from fides.api.ops.schemas.redis_cache import Identity
+from fides.api.ops.schemas.saas.shared_schemas import SaaSRequestParams
+from fides.api.ops.service.connectors import get_connector
+from fides.api.ops.service.privacy_request.request_runner_service import (
+    build_consent_dataset_graph,
+)
+from fides.api.ops.task import graph_task
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_mailchimp_transactional
+def test_mailchimp_transactional_connection_test(
+    mailchimp_transactional_connection_config,
+) -> None:
+    get_connector(mailchimp_transactional_connection_config).test_connection()
+
+
+@pytest.mark.integration_saas
+@pytest.mark.integration_mailchimp_transactional
+@pytest.mark.asyncio
+async def test_mailchimp_transactional_consent_request_task(
+    db,
+    consent_policy,
+    mailchimp_transactional_connection_config,
+    mailchimp_transactional_dataset_config,
+    mailchimp_transactional_identity_email,
+) -> None:
+    """Full consent request based on the Mailchimp Transactional (Mandrill) SaaS config"""
+
+    privacy_request = PrivacyRequest(
+        id=str(uuid4()),
+        consent_preferences=[{"data_use": "advertising", "opt_in": False}],
+    )
+
+    identity = Identity(**{"email": mailchimp_transactional_identity_email})
+    privacy_request.cache_identity(identity)
+
+    dataset_name = "mailchimp_transactional_instance"
+
+    v = await graph_task.run_consent_request(
+        privacy_request,
+        consent_policy,
+        build_consent_dataset_graph([mailchimp_transactional_dataset_config]),
+        [mailchimp_transactional_connection_config],
+        {"email": mailchimp_transactional_identity_email},
+        db,
+    )
+
+    assert v == {
```

Comment:
perhaps it's going overboard for this stage, but we should be able to actually reach out to mailchimp to verify that the consent request had the intended affect?

generally we've been trying to do that in our saas tests when possible, it just gives an extra level of certainty that things are working as expected. but there are definitely pros/cons to doing it, and it's also not a hard requirement for the beginning stages of implementing this functionality.

Reply:
hmm good point it does look like there's a POST /allowlists/list I can try to connect to that can be filtered on email.  I don't think we'll be able to verify this for every connector, but I I should be able to get something working here.  Also, do you know how to unskip saas tests in CI here?

Reply:
oh, do you just need to toggle the `run unsafe ci checks` label?

Reply:
I'll try that again when I push up this next round of changes, thanks!

Reply:
OK test added, the POST /allowlists/list endpoint worked well for verifying.